installer/pkg/cmd/gcp_broker.go

```diff
@@ -221,7 +221,7 @@ func getOrCreateGCPServiceAccount(name, email string) error {
 	_, err := gcp.GetServiceAccount(email)
 	if err != nil {
 		// TODO(droot): distinguish between real error and NOT_FOUND error
-		err = gcp.CreateServiceAccount(name, "Service Catalog GCP Broker Service Account")
+		err = gcp.CreateServiceAccount(name, "Google Cloud Platform Service Broker")
```

Comment:
I check the display name of the service accounts and found they always end with "service account".